src/components/forms/reply-comment-form/reply-comment-form.js

```diff
@@ -0,0 +1,200 @@
+import { useStyles } from './reply-comment-form.styles';
+import { BackButton, SaveButton } from '../../buttons';
+import CheckboxOptions from '../../checkbox-options';
+import { config } from '../../../configs';
+import {
+  addReplyComment,
+  updateReply
+} from '../../../redux/comments/comments.actions';
+import { showErrorSnackbar } from '../../../redux/snackbar/snackbar.actions';
+import { closeDialog } from '../../../redux/dialog-window/dialog-window.actions';
+import useSuccessSnackbar from '../../../utils/use-success-snackbar';
+
+const {
+  REPLY_COMMENT_VALIDATION_ERROR,
+  REPLY_COMMENT_ERROR_MESSAGE,
+  MAX_LENGTH_MESSAGE
+} = config.replyCommentErrorMessages;
+const { SAVE_MESSAGE, SAVE_CHANGES } = config.messages;
+
+const { SAVE_TITLE } = config.buttonTitles;
+const { pathToCommentsEdit } = config.routes;
+
+const ReplyCommentForm = ({
+  reply,
+  isEdit,
+  commentId,
+  adminId,
+  adminReply
+}) => {
+  const styles = useStyles();
+  const dispatch = useDispatch();
+
+  const history = useHistory();
+  const { openSuccessSnackbar } = useSuccessSnackbar();
+
+  const replyCommentValidationSchema = Yup.object().shape({
+    replyText: Yup.string()
+      .min(2, REPLY_COMMENT_VALIDATION_ERROR)
+      .max(300, MAX_LENGTH_MESSAGE)
+      .required(REPLY_COMMENT_ERROR_MESSAGE),
+    showReplyComment: Yup.bool()
+  });
+
+  const { values, handleSubmit, handleChange, touched, errors, setFieldValue } =
+    useFormik({
+      validationSchema: replyCommentValidationSchema,
+      initialValues: {
+        replyText: reply.replyText || '',
+        showReplyComment: adminReply ? true : reply.showReplyComment || false
+      },
+      onSubmit: (data) => {
+        if (isEdit) {
+          updateReplyCommentHandler(reply._id, data, reply.refToReplyComment);
+        } else {
+          addReplyCommentHandler(data);
+        }
+      }
+    });
+
+  const addReplyCommentHandler = (data) => {
+    const addReplyForComment = () => {
+      dispatch(closeDialog());
+      dispatch(
+        addReplyComment({
+          id: adminId,
+          commentId,
+          replyCommentData: {
+            replyText: data.replyText,
+            showReplyComment: data.showReplyComment,
+            refToReplyComment: commentId,
+            answerer: adminId
+          }
+        })
+      );
+    };
+    openSuccessSnackbar(addReplyForComment, SAVE_MESSAGE, SAVE_CHANGES);
+  };
+
+  const updateReplyCommentHandler = (id, data, commentIdUpdate) => {
+    const updateReplyForComment = () => {
+      dispatch(closeDialog());
+      dispatch(
+        updateReply({
+          replyCommentId: id,
+          replyCommentData: {
+            replyText: data.replyText,
+            showReplyComment: data.showReplyComment
+          },
+          commentId: commentIdUpdate
+        })
+      );
+    };
+    openSuccessSnackbar(updateReplyForComment, SAVE_MESSAGE, SAVE_CHANGES);
+  };
+
+  function handleCommentClick() {
```

Comment:
write functions in one style only, like you should choose either regular functions with "function" syntax or far arrow functions with () => {} syntax

Reply:
fix